ansible/roles/dashmate/tasks/main.yml

```diff
@@ -269,6 +269,38 @@ Dashmate Config Changed: {{ is_dashmate_config_changed }}\n"
     chdir: '{{ dashmate_cwd }}'
   changed_when: build_dashmate_helper.rc == 0
 
+# In the seed mode, we are disabling some services that are not necessary to run on the node
```

Comment:
I believe DAPI and envoy are still required. DAPI Client connects to a seed node to get SML through DAPI. So only Drive is actually not needed on seed nodes but it's ok if it's running - it won't do anything and consequently won't produce any load.

Reply:
Ok, but I wonder how does it work now? DAPI on the seeds is not working for a long time in the mn_evo_services

Reply:
What is needed to get it working? Should we investigate that in this PR, or separately?

Reply:
DAPI currently fails as follows on seed nodes:
Dashmate node:
```
  1) Smoke
       DAPI
         All nodes
           seed-1
             should return data from Core:
     AssertionError: FetchError: invalid json response body at https://35.92.167.154:1443/ reason: Unexpected token u in JSON at position 0
      at Context.<anonymous> (test/smoke/dapi.js:79:20)
      at processImmediate (node:internal/timers:466:21)
```
```
  [Symbol(Response internals)]: {
    url: 'https://35.92.167.154:1443/',
    status: 503,
    statusText: 'Service Unavailable',
    headers: Headers { [Symbol(map)]: [Object: null prototype] },
    counter: 0
  }
```
Non-working mn_evo_services node:
```
  7) Smoke
       DAPI
         All nodes
           seed-3
             should return data from Core:
     AssertionError: ResponseError: request to https://18.246.62.131:1443/ failed, reason: write EPROTO 140119617955776:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:

      at Context.<anonymous> (test/smoke/dapi.js:79:20)
      at processImmediate (node:internal/timers:466:21)
```

Reply:
This will be resolved in a separate PR, since adding DAPI functionality to seed nodes is out of scope for this PR